MathLib/LinAlg/Eigen/EigenVector.h

```diff
@@ -65,6 +65,19 @@ class EigenVector final
         return _vec[rowId];
     }
 
+    /// get entries
+    std::vector<double> get(std::vector<IndexType> const& indices) const
+    {
+        std::vector<double> local_x;
+        local_x.reserve(indices.size());
```

Comment:
not now: allocating memory for each time (for each element) can degrade speed. need to profile later on